WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostPreviewFragment.java

```diff
@@ -53,14 +54,25 @@ public void onResume() {
         }
     }
 
-    public void loadPost() {
-        new LoadPostPreviewTask().executeOnExecutor(AsyncTask.THREAD_POOL_EXECUTOR);
+    @Override
+    public void onPause() {
+        super.onPause();
+
+        if (mLoadTask != null) {
+            mLoadTask.cancel(true);
+            mLoadTask = null;
+        }
     }
 
+    public void loadPost() {
+        if (mLoadTask == null) {
```

Comment:
If there's a load in progress I didn't think it was appropriate to send off another request. When the previous load completes it will set mLoadTask to null and allow another load to take place.